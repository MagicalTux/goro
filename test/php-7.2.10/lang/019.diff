--
     hey, this is a regular echo'd eval()
     hey, this is a function inside an eval()!
     
022+ Fatal error: Uncaught Error: 
     Parse error: syntax error, unexpected %s in %s019.php(12) : eval()'d code on line 1
