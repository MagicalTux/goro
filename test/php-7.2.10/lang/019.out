hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!
hey, this is a regular echo'd eval()
hey, this is a function inside an eval()!

Fatal error: Uncaught Error: 
Parse error: syntax error, unexpected T_EOF in /home/ronald/proj/goro/test/php-7.2.10/lang/019.php(12) : eval()'d code on line 1